code/style.css

```diff
@@ -1,4 +1,199 @@
+/* Universal selector: it applies to all of the elements*/
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: orange;
+}
+
+body {
+  background-color: orange;
+  min-height: 100vh;
+  min-width: 500px; /*To make the body not to shrink less than 500px*/
+}
+/* HEADER STYLING */
+header {
+  background-color: whitesmoke;
+  height: 10em;
+  width: 100%;
+  position: fixed;
+  top: 0;
+}
+
+nav {
```

Comment:
wouldn't it be better to use class instead of tag for styling?

Reply:
Yes, sure! I tried not to use many classes as I was making only a one-page site, just to make it more simple but probably is not the best practice.